### ISSUE TITLE
SDL2: SDL_WINDOWEVENT event identified as 'Unknown'

### ISSUE THREAD

**Issue by charlesej** [pygame/pygame#1222]
The following code is used to show the `SDL_WINDOWEVENT` event is identified as 'Unknown'.

```python
import pygame

pygame.display.init()
pygame.display.set_mode((100, 100))
running = True

while running:
    for e in pygame.event.get():
        print(e)
        if e.type == pygame.QUIT:
            running = False

print()
print(0x200, pygame.event.event_name(0x200))
print(0x201, pygame.event.event_name(0x201))
pygame.display.quit()
```

This output shows the events that are generated as the window is created and then closed via clicking the 'X'. (Note: a window of size of (120, 100) is created when the requested size was (100, 100) - see issue #993).
```
pygame 2.0.0.dev3 (SDL 2.0.10, python 3.7.2)
Hello from the pygame community. https://www.pygame.org/contribute.html
<Event(40961-VideoResize {'size': (120, 100), 'w': 120, 'h': 100})>
<Event(40960-ActiveEvent {'gain': 1, 'state': 1})>
<Event(770-TextEditing {'text': '', 'start': 0, 'length': 0, 'window': None})>
<Event(40960-ActiveEvent {'gain': 1, 'state': 0})>
<Event(1024-MouseMotion {'pos': (60, 38), 'rel': (0, 0), 'buttons': (0, 0, 0), 'window': None})>
<Event(40962-VideoExpose {})>
<Event(1024-MouseMotion {'pos': (60, 37), 'rel': (0, -1), 'buttons': (0, 0, 0), 'window': None})>
<Event(1024-MouseMotion {'pos': (89, 22), 'rel': (29, -15), 'buttons': (0, 0, 0), 'window': None})>
<Event(1024-MouseMotion {'pos': (119, 0), 'rel': (69, -32), 'buttons': (0, 0, 0), 'window': None})>
<Event(40960-ActiveEvent {'gain': 0, 'state': 0})>
<Event(40960-ActiveEvent {'gain': 0, 'state': 1})>
<Event(770-TextEditing {'text': '', 'start': 0, 'length': 0, 'window': None})>
<Event(40960-ActiveEvent {'gain': 1, 'state': 1})>
<Event(512-Unknown {'event': 14, 'window': None})>
<Event(256-Quit {})>

512 Unknown
513 SysWMEvent
```

Reference for enums values: [SDL_events.h](https://www.libsdl.org/tmp/SDL/include/SDL_events.h)
```c
    /* Window events */
    SDL_WINDOWEVENT    = 0x200, /**< Window state change */
    SDL_SYSWMEVENT,             /**< System specific event */
```

System details:
- os: windows 10 (64bit)
- python: 3.7.2 (64bit)
- pygame: 2.0.0.dev3 (SDL: 2.0.10) at 7a5ed0b864645e7c325d80aa8866e837559c7a1c

**Comment by Bottersnike** [comment]
This should be closed in favour of #1223

**Comment by charlesej** [comment]
This is a different issue than #1223. For this issue the `SDL_WINDOWEVENT` is missing from the `_pg_name_from_eventtype()` function. For issue #1223 the drop event ids are there but are ifdef'd out.

```c
#ifdef SDL_DROPTEXT
        case SDL_DROPTEXT:
            return "DropText";
        case SDL_DROPBEGIN:
            return "DropBegin";
        case SDL_DROPCOMPLETE:
            return "DropComplete";
#endif
```
